src/traces/box/plot.js

```diff
@@ -102,6 +102,7 @@ function plotBoxAndWhiskers(sel, axes, trace, t) {
 
     var fn;
     if(trace.type === 'box' ||
+        trace.type === 'candlestick' ||
```

Comment:
Perhaps `trace.type !== 'violin' || trace.box`? NBD either way, we'd of course figure it out pretty quick if we add any other trace type that reuses box drawing.